docs/user_guide/quick_start.rst

```diff
@@ -192,6 +192,8 @@ Simulation results can be plotted with ``matplolib`` using the
     ax.legend()
     ax.loglog()
     ax.set_title("Logical CNOT Error Rate")
+    ax.set_xlabel("Physical Error Rate")
+    ax.set_ylabel("Logical Error Rate")
     plt.show()
```

Comment:
Interestingly, if we make changes to a cached page, they do not show up. We will have to delete the cache for them to show up and be cached again. 

https://tqec.github.io/tqec/pull/592/user_guide/quick_start.html#plot-the-results

Reply:
That is not normal in my opinion. The cache should not impact sphinx process at all, except when `tqec` and `stim` are called. Unresolving to check that later.

Reply:
One potential idea to look after: wait a few minutes before checking the actual page, and empty the local browser cache to be sure you have the last online version. Changes to GitHub pages may take a few minutes to propagate, so we should check that.